tests/index.js

```diff
@@ -0,0 +1,27 @@
+const test  = require('tape')
+const tapSpec = require('tap-spec')
+// You may need to set another const for relevant JS files
+
+test ('initial', function (t) {
+  t.plan(1)
+  t.equal('a', 'a')
+})
+
+
+test('tell us what you are testing', function (t) {
```

Comment:
This test only passes because we're comparing 2 undefined variables. Let's comment this test out and be more explicit about the arrange, act, assert.